### ISSUE TITLE
How to hide buttons?

### ISSUE THREAD

**Issue by zeref-dragneel** [TeamNewPipe/NewPipe#8492]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

<img src="https://i.imgur.com/z42HOd5.png" height="500" /> <img src="https://i.imgur.com/s9dycdn.png" height="500" />

### Why do you want this feature?

I like the minimalism. And I don't need to use those functions either.

**Comment by SameenAhnaf** [comment]
For your second image, you might enable this toggle from `Settings`>`Video and Audio`>`Start main player on full screen`

**Comment by zeref-dragneel** [comment]
Sorry, but I want to watch in small screen. Hopefully the next version will be able to hide the buttons

**Comment by SameenAhnaf** [comment]
You can still use `Start playing on popup` by long press on any video. I thought you don't need video info tab (related videos/ comments/ description) as none of them was enabled.

**Comment by zeref-dragneel** [comment]
Thanks for answering.
That image is just an example. I still allow the comment tab to be displayed.

**Comment by zeref-dragneel** [comment]
More: My grandparents are old and are using NewPipe, so they are often mistakenly pressed on unnecessary buttons.
The problem is that there is no **voice search**, so finding the videos for my grandparents is a difficult one.

<img src="https://i.imgur.com/mtMupMY.png" height="150" />

**Comment by SameenAhnaf** [comment]
That was rejected by https://github.com/TeamNewPipe/NewPipe/issues/1913

**Comment by zeref-dragneel** [comment]
Yes, I saw it, so I didn't ask for more voice searches. I only requires hiding unnecessary buttons.
